2017-2018/GameEngine/vision/cv.py

```diff
@@ -1,13 +1,103 @@
+import numpy as np
+from pacbot.grid import grid
+from comm.pacmanState_pb2 import PacmanState
+import asyncio, cv2
 
-import asyncio, opencv
+FREQUENCY = 60
 
 class MovementProcessor:
-    def __init__(self, client, loop=None):
+    def __init__(self, client, edges=None, cam_number=0, loop=None):
         self.client = client
         self.loop = loop or asyncio.get_event_loop()
         self.loop.call_soon(self.tick)
+        self.edges = edges if not edges == None else {'l':342, 'r':1185,'u':-8,'d':749}
```

Comment:
I also simplified this in my commit, so it will discard any value that evaluates to False instead of just None, but since it appears the only other valid values are these dictionaries, that seems like an acceptable behavior.

Reply:
nice